### DISCUSSION TITLE
web3工作咨询

### DISCUSSION THREAD

**Discussion by zaocj** [openbuildxyz/mina-zkapp-bootcamp#73] [General]
对web3感兴趣，怎么样才能开启一份web3的工作？

**Comment by dethan3** [comment]
再添加一个问题：Web3 行业找工作和传统找工作的方法有没有什么不同？

**Reply by ginger-coder** [reply]
在国内的web3感觉，会一如既往的卷，国外会好点

**Reply by phoouze** [reply]
开发来说的话基本还是看技术能力，相对来说面试高效不太会有八股文

**Comment by ScarboroughCoral** [comment]
还有一个比较关心的问题：web3 工资 u 换 CNY 有没有最佳实践？

**Reply by jjeejj** [reply]
交易所 或者 场外。如果正好有朋友需要 那就更好了

**Reply by MartinYeung5** [reply]
其實不一定是u，有些是以美元或當地貨幣，
如果需要轉換，建議在當地合法的交易所進行轉換，或向所屬公司了解。

**Reply by weirdcola** [reply]
u -> $ -> ¥
5w 美金每人每年的额度

**Comment by phoouze** [comment]
这是 OpenBuild 出品的Web3技术播客中的一期，主要聊进入Web3开发需要了解哪些知识和经验；Web2和Web3开发有什么不一样。这期的嘉宾既有成功从Web2进入到Web3的OG们，也有深耕开发者教育的知名开发者。
https://mp.weixin.qq.com/s/kB8Kct0Rz6o0wZSK8D105g

**Comment by coderhyy** [comment]
对web3感兴趣，怎么样才能从 100 到 A8

**Reply by jjeejj** [reply]
100 和 A8 是什么梗

**Reply by logere** [reply]
8位数?

**Reply by MartinYeung5** [reply]
什麼意思? 第一次聽

**Reply by phoouze** [reply]
> 什麼意思? 第一次聽

资产规模A8

**Comment by MartinYeung5** [comment]
\

> 对web3感兴趣，怎么样才能开启一份web3的工作？

1. 可以通過hackathon，有些會有招聘活動，也有企業會在活動中挑選人才。
2. 可以自己做一些項目，積累經驗，這樣都有助找到Web3工作。

**Comment by zhengqiuwan** [comment]
怎么样才能开启一份web3的工作？更靠谱还是靠内推，我的经验是通过web3的线下活动，多认识一些行业人和项目方。开发者可以多参加技术大会和黑客松和hackerhouse，通过项目直接展示自己的技能，很多项目方也担心被开发rug，所以线下的真实见面靠谱很重要！

**Comment by lispking** [comment]
> 对web3感兴趣，怎么样才能开启一份web3的工作？

有兴趣可以看看最近的Web3转型课程，看看大牛们怎么说：https://openbuild.xyz/learn/challenges/2050911877
